Additional npm module is required to run this?

I am getting this error when I run this module. I am guessing this is also required by this module. 
```
Error: Cannot find module '@beyonk/async-script-loader'

```

--- a/package.json
+++ b/package.json
@@ -10,7 +10,6 @@
     "prepublishOnly": "npm run build"
   },
   "devDependencies": {
-    "@beyonk/async-script-loader": "^1.0.3",
     "@gen/rollup-plugin-generate-html": "^0.1.1",
     "rollup": "^1.16.7",
     "rollup-plugin-commonjs": "^9.3.4",
@@ -41,5 +40,7 @@
     "index.mjs",
     "index.js"
   ],
-  "dependencies": {}
+  "dependencies": {
+    "@beyonk/async-script-loader": "^1.0.3"
+  }
 }

